cli/kaos_cli/utils/validators.py

```diff
@@ -8,7 +8,77 @@
 import textdistance
 from kaos_cli.exceptions.exceptions import MissingArgumentError
 
-from ..constants import KAOS_STATE_DIR, DOCKER, MINIKUBE, KAOS_TF_PATH
+from ..constants import KAOS_STATE_DIR, DOCKER, MINIKUBE, KAOS_TF_PATH, TF_STATE
+
+
+class EnvironmentState:
+    """
+    This class is used to validate existence of already built environmnets
+    """
+    def __init__(self):
+        self.cloud = None
+        self.env = None
+        self.build_dir = None
+        self.tf_state_path = None
+        self.if_build_dir_exists = None
+        self.if_tfstate_exists = None
+
+    @classmethod
+    def validate_build_states(cls, cloud, env) -> "EnvironmentState":
```

Comment:
Since this function only reads the environment and initializes an `EnvironmentState` instance, I would recommend renaming it to: `read_environment_state` or simply `read` since it's a static method and the meaning of the name can be semantically derived as so: `EnvironmentState.read()`.

The reason behind this is that the validator functions truly validate inputs and most of them issue a `sys.exit` in case the input data was invalid. However, this function doesn't.

Reply:
Yes, this method behaves more like an initializer for the environment state. Have renamed this as initialize().